meta-lmp-bsp/dynamic-layers/xilinx/recipes-bsp/device-tree/device-tree/kv260/0001-zynqmp-sm-k26-reva-enable-pwm-fan-for-fancontrol.patch

```diff
@@ -0,0 +1,35 @@
+From ce6972ebf88d50dccc5f140e4dba4bb83ce4c9ae Mon Sep 17 00:00:00 2001
+From: Ricardo Salveti <ricardo@foundries.io>
+Date: Fri, 30 Sep 2022 20:19:02 -0300
+Subject: [PATCH] zynqmp-sm-k26-reva: enable pwm-fan for fancontrol
+
+Upstream-Status: Inappropriate [lmp specific]
+
+Signed-off-by: Ricardo Salveti <ricardo@foundries.io>
+---
+ .../data/kernel_dtsi/2022.1/BOARD/zynqmp-sm-k26-reva.dtsi       | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/device_tree/data/kernel_dtsi/2022.1/BOARD/zynqmp-sm-k26-reva.dtsi b/device_tree/data/kernel_dtsi/2022.1/BOARD/zynqmp-sm-k26-reva.dtsi
+index ed38811..893767b 100644
+--- a/device_tree/data/kernel_dtsi/2022.1/BOARD/zynqmp-sm-k26-reva.dtsi
++++ b/device_tree/data/kernel_dtsi/2022.1/BOARD/zynqmp-sm-k26-reva.dtsi
+@@ -81,6 +81,7 @@
+ 	};
+ 	pwm-fan {
+ 		compatible = "pwm-fan";
++		status = "okay";
```

Comment:
`status = "okay"` is not needed - this value is dafault.

Reply:
Indeed, updated.

Reply:
Now `pwm-fan` got removed at all :)

I mean it can be like
```
pwm-fan {
 		compatible = "pwm-fan";
 }
 ```